wrapper.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-node usr/azure-integration/examples/simple.js
+node usr/azure-integration/bridge.js
 printf 'INTEGRATION PROCESS FATAL ERROR: %s\n' "$?"
```

Comment:
Isn't it already the case @johanstokking ? the bridge is an infinite loop. The only way to reach the print is when a hard exit happened which makes us exit the bridge program. Then I print the fatal error message and print the status